bigquery/google/cloud/bigquery/__init__.py

```diff
@@ -28,3 +28,6 @@
 from google.cloud.bigquery.dataset import Dataset
 from google.cloud.bigquery.schema import SchemaField
 from google.cloud.bigquery.table import Table
+from google.cloud.bigquery._helpers import ArrayQueryParameter
```

Comment:
Shouldn't these be at the top. I would think that `bigquey._helpers` would be before `bigquery.client`.